app/main/spawn.js

```diff
@@ -13,38 +13,60 @@ specific language governing permissions and limitations under the License.
 import { spawn } from 'child_process';
 import path from 'path';
 import * as endpoints from '../shared/constants/endpoints';
+import BUILD_CONFIG from '../../build-config';
 
-const DB_PATH = path.join(__dirname, '..', '..');
-const SERVICES_PATH = path.join(__dirname, '..', 'services');
+const ROOT = path.join(__dirname, '..', '..');
+const DB_PATH = ROOT;
+const SERVICES_PATH = path.join(ROOT, 'lib', 'services');
 const UA_SERVICE_BIN = path.join(SERVICES_PATH, 'user-agent-service', 'bin', 'user-agent-service');
 const CONTENT_SERVICE_PATH = path.join(SERVICES_PATH, 'content-service', 'index.js');
 
 // @TODO Use something like `forever` or `pm2` to ensure that this
 // process remains running
 // @TODO Will `node` be accessible via path on users machines that don't
 // have node installed as an engineer?
-export function startUserAgentService(userAgentClient) {
+
+const children = [];
+
+export function startUserAgentService(userAgentClient, options = {}) {
   const port = endpoints.UA_SERVICE_PORT;
   const host = endpoints.UA_SERVICE_ADDR;
   const version = endpoints.UA_SERVICE_VERSION;
 
-  spawn('node', [UA_SERVICE_BIN,
+  children.push(spawn('node', [UA_SERVICE_BIN,
     '--port', port,
     '--db', DB_PATH,
     '--version', version,
     '--content-service', endpoints.CONTENT_SERVER_ORIGIN,
   ], {
     detached: true,
-    stdio: ['ignore'],
-  });
+    // Careful passing in options.verbose as true, as this stops the services from
+    // outliving the their original parents, which we want to allow in the UAS case.
+    stdio: options.verbose ? ['ignore', process.stdout, process.stderr] : ['ignore'],
```

Comment:
The stdio options breaks Windows according to @mossop which was already reverted in the linked commit, so we should comment on that, and these changes would break Windows